proj-solar-system.rb

```diff
@@ -0,0 +1,101 @@
+
+
+
+class SolarSystem
+
+ def initialize (planets)
+  @planets = planets
+ end
+
+def printPlanet()
```

Comment:
This method doesn't actually **print** anything.  Instead it returns the planet details.